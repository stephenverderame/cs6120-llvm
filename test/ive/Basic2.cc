// NOLINTBEGIN
#include <cstdio>
int main()
{
    int m = 20;
    for (int i = 5000; i >= m; i -= m) {
        printf("%d\n", i);
    }
}
// NOLINTEND