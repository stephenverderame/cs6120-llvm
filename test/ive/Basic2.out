Basic IV:   %3 = phi i32 [ 5000, %0 ], [ %5, %2 ]
Basic IV:   %5 = add nsw i32 %3, -20
